client/src/app/components/stardust/TruncatedId.scss

```diff
@@ -0,0 +1,9 @@
+.truncated-id {
+    span {
+        width: 100px;
```

Comment:
Maybe we should consider not limiting the `truncated-id` to `width: 100px;` rather we could have truncated-id be width:100% and add `max-width` on the container element or something along these lines. That way the truncated-id would shrink or get larger base on the containers width. Wdyt?